docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2641,291 +2641,213 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0017E":"Identifier doesn't match one found in pluginDefinition: %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
+### ZWED0017E 
 
-
-
-### ZWED0018E":"No plugin type found, skipping
+  Identifier doesn't match one found in pluginDefinition: _pluginIdentifier_
 
   **Reason:**
 
-  TODO
+  The identifier found in the plugin reference doesn't match the one specified in the pluginDefinition.json
 
   **Action:**
 
-  TODO
-
-
-
-### ZWED0019E":"Plugin already registered
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
+  Check if identifier found is the same one as intended (typo perhaps?). If not, edit pluginDefinition.json or turn on App server debugging for the bootstrap logger (_zsf.bootstrap) and check if the plugin reference found in code ZWED0044I is correct
```

Comment:
in v2, these plugin files are automatically managed so, it's better to just delete the file and restart zowe to see if the issue persists, and if it does, then increase that debugging and send the logs to support.